stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Extensions.kt

```diff
@@ -0,0 +1,76 @@
+package io.getstream.chat.ui.sample.common
+
+import android.app.Activity
+import android.content.Context
+import android.view.inputmethod.InputMethodManager
+import android.widget.EditText
+import android.widget.Toast
+import androidx.annotation.IdRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavController
+import androidx.navigation.NavDirections
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.ui.sample.R
+
+var User.image: String
+    get() = extraData["image"] as String
+    set(value) {
+        extraData["image"] = value
+    }
+
+var User.name: String
+    get() = extraData["name"] as String
+    set(value) {
+        extraData["name"] = value
+    }
```

Comment:
It is already implemented in our `stream-chat-android-client` 
It shouldn't be needed here
```suggestion
```